agent/agent_endpoint.go

```diff
@@ -437,8 +437,19 @@ func (s *HTTPServer) AgentForceLeave(resp http.ResponseWriter, req *http.Request
 		return nil, acl.ErrPermissionDenied
 	}
 
+	//Check the value of the prune query
+	var prune bool
```

Comment:
Elsewhere, like in `agent/snapshot_endpoint.go` we allow for `?stale` to imply truthiness in a much simpler block of code (adapted for your use case):
```
_, prune := req.URL.Query()["prune"]
```